### ISSUE TITLE
Flow can't infer types from schema

### ISSUE THREAD

**Issue by mkatrenik** [andreypopp/validated#3]
Hi,
I'm not sure where's the problem, but when I use **validated** in my app, flow is unable to infer types from schema, I would just get `any`. But when I clone this repo & run flow there - everything works. So I wonder, could be some problem in compilation during publishing to npm? Has anyone else this problem?

flow@0.33.0

**Comment by andreypopp** [comment]
Can you post an example schema?

**Comment by mkatrenik** [comment]
Even if I'm using schema from `src/__tests__/index-test.js.flow` like:
```
  let schema = s.object({
    name: s.string,
    age: s.number,
  });
```
 it doesn't work

**Comment by mkatrenik** [comment]
Also on line 22 it says "unused suppresion"

<img width="568" alt="flow" src="https://cloud.githubusercontent.com/assets/887117/19089332/1f9ebef0-8a7a-11e6-9056-34651b5cd1e2.png">

**Comment by andreypopp** [comment]
Fixed in 0.8.1.

**Comment by mkatrenik** [comment]
Thanks!
